tasmota/tasmota_xdrv_driver/xdrv_02_9_mqtt.ino

```diff
@@ -1190,12 +1190,9 @@ void MqttReconnect(void) {
     MqttNonTLSWarning();
   }
 #ifdef USE_MQTT_AWS_IOT
-  // re-assign private keys in case it was updated in between
+  // re-assign private key in case it was updated in between
   if (Mqtt.mqtt_tls) {
     if ((nullptr != AWS_IoT_Private_Key) && (nullptr != AWS_IoT_Client_Certificate)) {
```

Comment:
As indicated in comment, this would break any existing client cert configuration where user/password have been left to default values. AWS IOT will refuse connection if you send non-empty user/password

Reply:
ok then I think it will be useful to add a new define to let use custom certificate with mosquitto acl compatibility
What do you think about USE_MQTT_CUSTOM_CERT?

Reply:
OT: I saw that web doesn't accept ';' char in password. Maybe some validation is required?